openfl/interface/workspace.py

```diff
@@ -423,89 +340,132 @@ def _get_dir_hash(path):
     return hash_
 
 
+# Commands for workspace packaging and distribution
+# -------------------------------------------------
+
+
+@workspace.command(name="export")
+def export_() -> str:
+    """
+    Exports the OpenFL workspace (in current directory)
+    to an archive.
+
+    \b
+    The archive contains the following files/dirs copied as-is:
+        - `src`: All experiment source code.
+        - `plan`: The FL plan directory.
+        - `save`: Model initial weights.
+        - `requirements.txt`: Package list required for the experiment.
```

Comment:
AFAIK, at the moment when the model owner exports the workspace, he is in possession of all the public certs (both client and server side). Why couldn't we include them all here for convenience?

PS: as the certs weren't included even in the original export implementation, this comment could be addressed in a separate PR

Reply:
Exporting a workspace happens before possession of certificates. It is the equivalent of "building a container image of all the computation that has to be done". This image is common for all participants.

Certificates and datasets mounted as volumes, since they vary for each participant.

Reply:
Actually, given that they are public information, the entirety of the federation's PKI certificates (agg, col, CA cert chain, etc.) can be bundled together and shared across the board. This is what the Certificate Server does in OpenFL-Security, right?

Anyhow, this discussion goes beyond the scope of the PR - so feel free to ignore me haha 😄

Reply:
This is a good idea. I'm in for this, especially if there is potential to cut down steps!